AlicerceTests/Logging/LogItemJSONFormatterTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  LogItemJSONFormatterTests.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 04/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+@testable import Alicerce
+
+class LogItemJSONFormatterTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        Log.removeAllProviders()
+    }
+    
+    func testLogItemJSONFormatter() {
+        
+        let provider = StringProvider()
+        provider.minLevel = .verbose
+        provider.linefeed = ","
+        provider.formatter = LogItemJSONFormatter()
+        
+        Log.registerProvider(provider: provider)
+        Log.verbose(message: "verbose message")
+        Log.debug(message: "debug message")
+        Log.info(message: "info message")
+        Log.warning(message: "warning message")
+        Log.error(message: "error message")
+        
+        let jsonString = "[\(provider.output)]"
+        let jsonData = jsonString.data(using: .utf8)
+        
+        if let obj = try? JSONSerialization.jsonObject(with: jsonData!, options: .allowFragments) {
+            let arr = obj as? Array<Dictionary<String, Any>>
```

Comment:
prefer using `guard let arg = obj as?  [[String : Any]]` with `return XCFail()` to avoid force unwraps.